webpack/base.js

```diff
@@ -5,11 +5,11 @@ module.exports = () => {
   return {
     entry: {
       "login": [
-        "./src/apps/loyalty/containers/login/browser",
+        "babel-polyfill", "./src/apps/loyalty/containers/login/browser",
       ],
       "inquiries": [
-        "./src/apps/loyalty/containers/inquiries/browser",
-      ]
+        "babel-polyfill", "./src/apps/loyalty/containers/inquiries/browser",
```

Comment:
Our `Object.assign` call is invoked on nodejs, so that shouldn't be a problem. However I still the following error when running without polyfill:

<img width="100%" alt="screen shot 2017-04-21 at 11 43 18 am" src="https://cloud.githubusercontent.com/assets/386234/25285301/be0b48f2-2687-11e7-9ca1-c8e4c057996c.png">